packages/attribute-registry-process/test/db.test.ts

```diff
@@ -67,6 +77,7 @@ describe("database test", () => {
     config.eventStoreDbPort = postgreSqlContainer.getMappedPort(5432);
     config.readModelDbPort = mongodbContainer.getMappedPort(27017);
     attributes = ReadModelRepository.init(config).attributes;
+    tenants = ReadModelRepository.init(config).tenants;
     readModelService = readModelServiceBuilder(config);
```

Comment:
```suggestion
    const readModelRepository = ReadModelRepository.init(config);
    attributes = readModelRepository.attributes;
    tenants = readModelRepository.tenants;
    readModelService = readModelServiceBuilder(readModelRepository);
```

This is needed in order to get rid of the `WARNING` in the tests.
You will have to modify the `readModelServiceBuilder` too, take a look how it's done in the catalog package

Reply:
Updated, but the warning didn't come from there.
The warning was related to the duplicate postgres connection, because during the test we instantiate 2 connections: one manually and one inside `attributeRegistryServiceBuilder`.
To solve the warning, I refactored the `attributeRegistryServiceBuilder` so now it requires a postgres connection.